src/frontend/src/lib/components/carousel/Carousel.svelte

```diff
@@ -274,7 +279,7 @@
 	{#if nonNullish(slides) && slides.length > 1}
 		<div
 			class={`absolute bottom-2 right-0 flex justify-between px-3 ${controlsWidthStyleClass ?? 'w-full'}`}
-			transition:slide={SLIDE_PARAMS}
+			out:slide={SLIDE_PARAMS}
```

Comment:
Wondering - wasn't `transition` covering the `out` case?

Reply:
yes yes, i removed the `in` transition in this way: I never noticed but it was a bit strange if you replay it slowly eheh! not noticeable too much, but you never know